src/otx/algo/modules/conv_module.py

```diff
@@ -225,7 +226,7 @@ def __init__(
         self.init_weights()
 
     @property
-    def norm(self) -> str | None:
+    def norm_layer(self) -> nn.Module | None:
```

Comment:
Could you update docstring also?

Reply:
https://github.com/openvinotoolkit/training_extensions/pull/3741/commits/28837861e6f6151bef8608a940b020bc6ed22437